config/collectorConfig.js

```diff
@@ -31,6 +31,8 @@ if (heartbeatLatencyToleranceMillis < FLOOR) {
   heartbeatLatencyToleranceMillis = 1000 * LATENCY_DEFAULT;
 }
 
+const sslOnly = +pe.COLLECTOR_SSL_ONLY || false;
```

Comment:
We use the "+" symbol when we want to try to coerce the env var to numeric value.
In this case here, we would want to set sslOnly to true if the env var **string** was "true" (and probably also accept "True"|"TRUE", but not "False" or "FALSE" or "false" ...

So I would suggest something like this:

```
const requireSslToRemoteDataSource = false;
if (pe.COLLECTOR_REQUIRE_SSL_TO_REMOTE_DATA_SOURCE && pe.COLLECTOR_REQUIRE_SSL_TO_REMOTE_DATA_SOURCE.toLowerCase() === 'true') {
  requireSslToRemoteDataSource = true;
}
```

Reply:
Oh, we should also write this up in the README section "Securing Refocus" so that a Refocus admin knows how to do this.

Reply:
good point, thx

Reply:
Done